fus.c

```diff
@@ -832,6 +832,11 @@ resolve_all_solvables (Pool  *pool,
                       Id p = t.elements[j];
                       Solvable *s = pool_id2solvable (pool, p);
 
+                      /* Make sure to include the module into the pile even if
+                       * it doesn't contain any components (e.g, a meta-module)
```

Comment:
I would say "empty"

Reply:
@sgallagher gave out to me today for calling it "empty".

Reply:
please make it empty